webservices/tests/candidate_tests.py

```diff
@@ -1,96 +1,154 @@
             response['pagination'],
-            {'count': 1, 'page': 1, 'pages': 1, 'per_page': 1})
-        self.assertEqual(len(response['results']), 1)
+            {'count': 20, 'page': 1, 'pages': 1, 'per_page': 20})
+        # we are showing the full history rather than one result
+        self.assertEqual(len(response['results']), 20)
 
         result = response['results'][0]
         self.assertEqual(result['candidate_id'], 'H0VA08040')
-        self.assertEqual(result['form_type'], 'F2')
+        self.assertEqual(result['form_type'], 'F2Z')
         # @todo - check for a value for expire_data
         self.assertEqual(result['expire_date'], None)
-        self.assertEqual(result['load_date'], '1990-02-05 00:00:00')
-        self.assertResultsEqual(
-            result['name'],
-            {
-                "full_name": "MORAN, JAMES P. JR.",
-                "name_1": "JAMES P JR",
-                "name_2": "MORAN",
-                "other_names": [
-                    "MORAN, JAMES P JR", "MORAN, JAMES P",
-                    "MORAN, JAMES PATRICK", "MORAN, JAMES P  JR."]
-            })
-        address = {
-            "city": "ALEXANDRIA",
-            "expire_date": "2009-01-01",
-            "state": "VA",
-            "street_1": "311 NORTH WASHINGTON STREET",
-            "street_2": "SUITE 200L",
-            "zip": "22314"
-        }
-        self.assertTrue(address in result['mailing_addresses'])
-
-        self.assertEqual(2, len(result['elections']))
-        election = result['elections'][0]
-        primary_committee = election['primary_committee']  # tested separately
-        del election['primary_committee']
-        self.assertResultsEqual(
-            election,
-            {
-                # From office_mapping
-                "office": "H",
-                "district": "08",
-                "state": "VA",
-                "office_sought": "H",
-                "office_sought_full": "House",
-                # From party_mapping
-                "party": "DEM",
-                "party_affiliation": "Democratic Party",
-                # From status_mapping
-                "election_year": 2014,
-                "candidate_inactive": "Y",
-                "candidate_status": "C",
-                "incumbent_challenge": None,
+        # most recent record should be first
+        self.assertEqual(result['load_date'], '2013-04-24 00:00:00')
+        self.assertResultsEqual(result['name'], 'MORAN, JAMES P. JR.')
+        #address
+        self.assertEqual(result['address_city'], 'ALEXANDRIA')
+        self.assertEqual(result['address_state'], 'VA')
+        self.assertEqual(result['address_street_1'], '311 NORTH WASHINGTON STREET')
+        self.assertEqual(result['address_street_2'], 'SUITE 200L')
+        self.assertEqual(result['address_zip'], '22314')
+        self.assertEqual(16, len(result['committees']))
+        # office
+        self.assertResultsEqual(result['office'], 'H')
```

Comment:
I've worked on this a bit, and I need to port some more assert features over from Hourglass, but since the sprint is ending, let's go ahead and get this merged. We need to make a separate pass on the tests. 

And I missed answering your question last time, @LindsayYoung ... yes, basically, we should just test the entire JSON structure as a whole, unless we are doing something specific. It's way more straightforward that way.